tests/cases/resource_test.py

```diff
@@ -455,10 +455,26 @@ def testGetResourceByPath(self):
                 str(resp.json['_id']), str(item['_id']))
 
         # test bogus path
+        # test is not set
         resp = self.request(path='/resource/lookup',
                             method='GET', user=self.user,
-                            params={'path': '/bogus/path'})
+                            params={'path': '/collection/bogus/path'})
```

Comment:
Can you change this back to just `/bogus/path`? It was covering a branch that is no longer covered after this change.